### DISCUSSION TITLE
Problema importazione cattedre

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#751] [Richieste di aiuto]
@trinko  Buongiorno, siamo ormai agli sgoccioli prima dell'inizio del nuovo anno scolastico ,pertanto stavo effettuando l'importazione delle diverse cattedre dei docenti e, non appena carico il file csv ,mi accetta solo le cattedre dalla cella numero 1 alla cella 89 ,oltre non va ,lasciandomi il seguente messaggio di errore : Errore Valore obbligatorio mancante nel record 90.Non capisco il motivo eppure ho ricontrollato tutto prima di effettuare le importazioni .Dal momento che si tratta di dati sensibili ,mi dica lei se è opportuno che le mani via mail il file in modo da correggere l'errore ,sempre se è possibile .Grazie

**Comment by trinko** [comment]
Devi controllare cosa ha di particolare il record 90, corrispondente alla riga 91 del file CSV (l'intestazione è la prima riga).
In generale l'errore indicato è dovuto alla mancanza della username del docente, del nome della classe o del nome della materia.

**Reply by sempliciniluigi89** [reply]
Ho controllato e non mi sembra ci siano errori ,addirittura ho provato a cancellare  il record 91 ,a quel punto permette di caricare un altro tot di cattedre, ma il problema si presenta sempre, anche con altri docenti . Se prima era il record 90 il problema ,eliminato questo ,il problema è il record 103,110,113 etc... e non capisco il motivo ,dal momento che tutti i dati sono stati registrati prima in piattaforma .Essendo tante cattedre non posso prima caricare il file csv di volta in volta scoprire il record sbagliato ed eliminarlo .

**Comment by trinko** [comment]
Nei record che danno problemi, ci sono caratteri speciali come lettere accentate?

**Comment by sempliciniluigi89** [comment]
No

Il giorno sab 7 set 2024 alle ore 10:54 Antonello Dessì <
***@***.***> ha scritto:
